cypress/integration/languageTest.spec.js

```diff
@@ -2,7 +2,7 @@
 
 describe("Language Test", () => {
     it("Deutsch", () => {
-        cy.visit(Cypress.env('TEST_SERVER') + `/?to=yucatan`)
+        cy.visit('localhost:3000')
```

Comment:
Why changing this to `localhost:3000` again? TEST_SERVER is set in https://github.com/Plant-for-the-Planet-org/planet-donations/blob/105110bce3ced85ed92ca197333fcb4fcea954f5/.github/workflows/cypress.yml#L38

Reply:
I'm not sure about this. I'll have to read. But It were failing tests when I tried running on my machine. Any which ways the visited ink is `localhost:3000` so I'm unable to see the difference.

Reply:
It might be that `cy.visit('localhost:3000')` works, but `cy.visit('http://localhost:3000')` has problems, but I can't explain why this should happen. Trying to change `http://localhost:3000` towards `localhost:3000` in cypress.env.json would help then.

Reply:
I changed back the test towards using an `cypress.env.json` file storing local secrets now. Every developer should have this file in his development environment if he wants to run the tests locally.